js/whitebit.js

```diff
@@ -95,6 +100,55 @@ module.exports = class whitebit extends Exchange {
                         'trades/{market}',
                     ],
                 },
+                'publicV4': {
+                    'get': [
```

Comment:
Public was not in the scope but I've decided to add it here just to make it accessible trough ccxt